From 4a51b79bf680b77687dc374ab4f992c0104e19fb Mon Sep 17 00:00:00 2001
From: Michael Catanzaro <mcatanzaro@gnome.org>
Date: Mon, 6 Apr 2020 19:20:30 -0500
Subject: [PATCH] Update version number in meson.build

This was forgotten before the 2.1.0 release.

It seems the 2.1.0 release tarballs were not generated with 'meson dist'. Using 'meson dist' to build and validate the tarball would allow immediately noticing such issues.
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index dc677ef2..f15251b3 100644
--- a/meson.build
+++ b/meson.build
@@ -1,5 +1,5 @@
 project('openh264', ['c', 'cpp'],
-  version : '2.0.0',
+  version : '2.1.0',
   meson_version : '>= 0.47',
   default_options : [ 'warning_level=1',
                       'buildtype=debugoptimized' ])
-- 
2.26.1.windows.1

